src/components/Organisms/MarketingPreferencesDS/assets/CR_@.svg

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Generator: Adobe Illustrator 24.3.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
```

Comment:
I use `svgo` as a global package to optimize svgs.. https://www.npmjs.com/package/svgo